File: HW07_CS/Cylinder.cs
namespace HW07
{
    class Cylinder{
        public double Radius { get; set; } = 1.0;
        public double Length { get; set; } = 1.0;

        public double Area => 2 * Math.PI * Radius * (Radius + Length);

        public double Volume => Area * Length;

        public void CylinderInfo(){
            System.Console.WriteLine($"Radius: {Radius:N2}");
            System.Console.WriteLine($"Length: {Length:N2}");
            System.Console.WriteLine($"Area: {Area:N2}");
            System.Console.WriteLine($"Volume: {Volume:N2}");
        }
    }
}

File: HW07_PY/Cylinder.py
import math

class Cylinder:
    def __init__(self, radius=1.0, length=1.0):
        self.__radius = radius
        self.__length = length

    @property
    def radius(self):
        return self.__radius

    @radius.setter
    def radius(self, value):
        self.__radius = value

    @property
    def length(self):
        return self.__length

    @length.setter
    def length(self, value):
        self.__length = value

    def area(self):
        return 2 * math.pi * self.__radius * (self.__radius + self.__length)

    def volume(self):
        return self.area() * self.__length

    def CylinderInfo(self):
        print(f"Radius: {self.radius}")
        print(f"Length: {self.length}")
        print(f"Area: {self.area():.2f}")
        print(f"Volume: {self.volume():.2f}")